crates/uv/tests/pip_compile.rs

```diff
@@ -127,24 +127,23 @@ fn missing_requirements_in() {
 fn missing_venv() -> Result<()> {
     let context = TestContext::new("3.12");
     context.temp_dir.child("requirements.in").touch()?;
-    fs_err::remove_dir_all(context.temp_dir.child(".venv").path())?;
+    fs_err::remove_dir_all(context.venv.path())?;
 
     uv_snapshot!(context.filters(), context.compile()
             .arg("requirements.in"), @r###"
-    success: false
-    exit_code: 2
+    success: true
+    exit_code: 0
```

Comment:
This test needs to fail

Reply:
I don't think this is correct. Why would it fail? You don't need a virtual environment to run `pip compile`.

Reply:
I see, that's a different test than i thought it was